src/shared/services/shortcut-action.service.ts

```diff
@@ -215,6 +215,12 @@ export class ShortcutActionService {
     this.domUtilService.focusAndSelectFirstInputChildById(path);
   }
 
+  private clearFocus(path: string) {
```

Comment:
move to dom-utils service, name it `blurFirstInputChildById`. Also it would be nice if you write in multiple lines, 
```
let el = doc...byId(..);
let firstInput = el.querySelector(...) as HTMLInputElement;
firstInput.blur();